cpp/src/arrow/filesystem/azurefs_test.cc

```diff
@@ -376,6 +376,10 @@ culpa qui officia deserunt mollit anim id est laborum.
     return s;
   }
 
+  std::string RandomFileName(RNG &rng) { return RandomChars(10, rng); }
+
+  std::string RandomFileNameExtension(RNG &rng) { return RandomChars(3, rng); }
```

Comment:
No, it generates the filename only, sans extension.  I suppose that it could be modified to generate XXXXXXX.ZZZ however, if that would be preferred.  See commentary above.  I could just place everything inline and delete these functions if that is the route that is recommended.

Reply:
Combined.